src/components/button/button.module.css

```
@@ -0,0 +1,9 @@
+.button {
+  border: none;
+  background-color: tomato;
+  border-radius: 3px;
+  color: white;
```

```suggestion
  color: #fff;
```

Creo que se deberia de usar color HEX